### DISCUSSION TITLE
How are the conversions ordered?

### DISCUSSION THREAD

**Discussion by StevenDTX** [jlesage/docker-handbrake#356] [Q&A]
I added a bunch of videos to a single watch folder. Several hours later, I added a few more to the same folder.  Those recently added videos completed somewhere in the middle, and not in the end.

I honestly don't care when things complete, just kind of curious when a conversion should start.

**Comment by jlesage** [comment]
When starting to process the watch folder, all files to convert are stored in a list at the beginning. Thus, new files added to the watch folder while current ones are being processed should be handled after the list is processed.

This doesn't seem to match what you are describing. Maybe you can look at the container's log to see what is happening.
